test/util/card-link.js

```diff
@@ -0,0 +1,7 @@
+export default class CardLink{
+  constructor(key, type, linkType){
```

Comment:
`type`? Do you mean issueType? Since your third argument gives a description on what type of type it is, it will be helpful to reflect that in your second argument as well.